src/views/ConfigurationActionsView.vue

```diff
@@ -315,6 +316,7 @@
 <script setup lang="ts">
 import { computed, onMounted, reactive, ref } from 'vue'
 
+import JavascriptActionConfig from '@/components/configuration/JavascriptActionConfig.vue'
 import MavlinkMessageActionConfig from '@/components/configuration/MavlinkMessageActionConfig.vue'
```

Comment:
Since you have made the atomization for the  JavascriptActionConfig and MavlinkMessageActionConfig components, I think would be nice to also do the same with the 'HTTP Request Actions' part on this template.

Reply:
Definitely! Will do that now.